packages/commerce-sdk-react/src/provider.tsx

```diff
@@ -17,8 +32,52 @@ export interface CommerceApiProviderProps {
     currency: string
 }
 
+/**
+ * @internal
+ */
+export const CommerceApiContext = React.createContext({} as ApiClients)
+
+/**
+ * Initialize a set of Commerce API clients and make it available to all of descendant components
+ *
+ * @param props - Config values to initialize the Commerce API clients
+ * @returns Provider to wrap your app with
+ */
 const CommerceApiProvider = (props: CommerceApiProviderProps): ReactElement => {
-    return <></>
+    const {children, clientId, organizationId, shortCode, siteId, proxy} = props
+
+    const config = {
+        proxy,
+        headers: {},
+        parameters: {
+            clientId,
+            organizationId,
+            shortCode,
+            siteId
+        },
+        throwOnBadResponse: true
```

Comment:
`throwOnBadResponse` is the recommended behavior, and is only a flag because we forgot to implement before our first release. 😑 If there's a use case that needs to access the response object even on a bad response, it's available as a property on the error that is thrown.